features/mbedtls/inc/mbedtls/mbedtls_config_reader.h

```diff
@@ -0,0 +1,457 @@
+/*
```

Comment:
This file should be added to the Mbed TLS repository, and not to Mbed OS. It will be overridden next time Mbed TLS release is delivered to Mbed OS, or, to `features/mbedtls/platform/inc/`

Reply:
I agree with @RonEld that this file should not be in this folder. If you wanted it here, then it should be merged into the main Mbed TLS repository, BUT I do not think it belongs there because it looks very Mbed OS specific

Reply:
Good point. I'll find another place to put this file.